### ISSUE TITLE
Context of transfer learning / finetuning

### ISSUE THREAD

**Issue by alphadadajuju** [CalculatedContent/WeightWatcher#253]
Thank you for the amazing tool! When coping with small validation set where its scoring metric flutuates and being less reliable, I believe WeightWatcher certainly can serve as an additional means of monitoring model training. 

However, since the tool is completely test data independent, could you elaborate how we can interpret or leverage layer-wise alphas within the context of transfer learning / finetuning with pretrained models? For instance, when launching transfer learning, I noticed that my model's first 3-4 CNN layers quickly arrived to "over-trained" status (very small alphas) at the end of the **first epoch**. In addition, I also observed that the few **newly added dense layers** (for finetuning) quickly exited "under-trained" status within just a few epochs of training (e.g., 3 epochs). In both cases above, my model was far from being converged (according to loss curves and validation accuracy). Are these behavior expected? 

Many thanks again.

**Comment by charlesmartin14** [comment]
it's difficult to see what's going on without looking at your model in detail. join the discord channel, we can take a deep dive.

Briefly, however, what you're seeing is typical of the correlation flow in many models like VGG. The early layers content to be a little over fit because the heavy tell nature of the date of bleeds in very strongly

Broadly speaking, however, it sounds like you're architecture has a correlation flow issue

There are some remedies you can try compensate for this in architecture you're using . This includes:
 - Add layer specific regularization. Try turning down the regularization all the early layer
 - Adjusting the learning rate for each layer independently. Turn the learning rate down for layers that are over, fit and turn it up for those that are under his
- Add a residual connection from the overfit layer to the under fit layer. This will improve the correlation flow.

Also, we need to look at the size of your layers . Small layers may converge faster than larger layers because they have less parameters. You may need to adjust the layer sizes to avoid this overfitting.

**Comment by alphadadajuju** [comment]
Thank you very much for your insightful and detailed suggestions! I will try them out and also dig in a bit more into concepts of correlation flow that you brought up here and throughout your talks. 

Meanwhile, I can share a bit more information of my architecture here (and why WW may not be that indicative for my models). It was a non-standard ResNet10, which is already fairly small. When performing transfer learning on this **intact** architecture, WW reported normally (no over- or under-trained layers). 

Afterwards, I applied structural pruning (removal of CNN filters) to obtain a even smaller version of ResNet10, and then trained (not fine-tuned) this reduced architecture via transfer learning. This is where I made the observations posted in this issue. I think for starters, the reduced version of ResNet10 may simply be too small for an accurate WW assessment. Secondly, when I applied structural pruning by cutting the pretrained filters of my ResNet10, it may have destroyed certain **distrubution** of the weights leveraged by WW (since the scale and distribution of the pretrained weights were originally obtained by training on the complete ResNet10). 

These are just my conjecture and I will try to investigate more deeply. Many thanks again.

**Comment by charlesmartin14** [comment]
you are welcome
